dagql/idtui/testdata/TestTelemetry/TestGolden/viztest/broken/broken

```diff
@@ -12,6 +12,8 @@ Expected stderr:
 │ ! input: module.withSource.initialize failed to initialize module: failed to call module "broken" to get functions: call constructor: process "go build -ldflags -s -w -o /runtime ." did not complete successfully: exit code: 1
 │ │ $ ModuleSource.resolveFromCaller: ModuleSource! X.Xs CACHED
 │ │ │ ✔ upload /XXX/XXX/XXX from XXXXXXXXXXX (client id: XXXXXXXXXXX, session id: XXXXXXXXXXX) (include: XXXXXXXXXXX) (exclude: XXXXXXXXXXX) X.Xs
+│ │ │ │ ✔ filesync X.Xs
+│ │ │ │ │ ✔ copy X.Xs
```

Comment:
What were you thinking of in terms of further improvements, just more spans? I added these mostly because they are useful for knowing why a filesync is slow, but they seemed useful enough for users potentially that they may as well be non-internal.

> Side note: is the (client id: XXXXXXXXXXX, session id: XXXXXXXXXXX) part still useful?

I'd say yes, though admittedly mostly for debugging as a dagger developer, I don't think they are useful/meaningful to end users.

Reply:
> What were you thinking of in terms of further improvements, just more spans?

Oh nothing in particular, these seem OK as-is - I was just wondering things like 'should more info be in the name' and 'do we need two spans' - but eyeballing it irl, there's a clear time gap between the `filesync` and the `copy` starting, so it seems worth the breakdown.